app/services/s_f_import.rb

```diff
@@ -185,9 +185,15 @@ def get_inventory(job, sales, products, bundles)
       end
     end
     inventory = []
+    skus = filter_skus(skus)
```

Comment:
I'd probably call this `filtered_skus`. In general I don't like reusing variables since it could get confusing as to what's in that variable at a certain line of code.

(I'm aware that this is happening in other places in the code as well.) :)